website/gulpfile.babel.js

```diff
@@ -99,8 +100,17 @@ gulp.task("copy", () =>
         "utf8",
         content =>
           new Promise((resolve, reject) => {
-            const contributors = JSON.parse(content);
-            resolve(yaml.dump({ contributors: contributors.contributors }));
+            const contributors = JSON.parse(content).contributors.map(c => {
+              const avatar = url.parse(c['avatar_url'], true);
+              avatar.search = undefined; // Override so that we can use `avatar.query`.
+              if (avatar.hostname.endsWith('githubusercontent.com')) {
```

Comment:
Do we even need to check this? I'm guessing the avatars are always going to be from GitHub, and even if not, a query parameter shouldn't break anything.

Reply:
Horse a piece, I'm fine with leaving it. It's explicit at least.